lib/internal/test_runner/test.js

```diff
@@ -7,15 +7,20 @@ const {
   Number,
   ReflectApply,
   SafeMap,
-  PromiseRace,
   SafePromiseAll,
+  SafePromiseRace,
+  PromiseResolve,
```

Comment:
Can we keep that list in alphabetical order?

Reply:
Yep, IMO this kind of stuff should either be linted or not enforced at all